src/components/TokenSelect/index.tsx

```diff
@@ -13,16 +15,27 @@ import { ThemedText } from 'theme'
 
 import Column from '../Column'
 import Dialog, { Header } from '../Dialog'
-import { inputCss, StringInput } from '../Input'
 import Row from '../Row'
 import Rule from '../Rule'
 import NoTokensAvailableOnNetwork from './NoTokensAvailableOnNetwork'
 import TokenButton from './TokenButton'
 import TokenOptions, { TokenOptionsHandle } from './TokenOptions'
 import TokenOptionsSkeleton from './TokenOptionsSkeleton'
 
-const SearchInput = styled(StringInput)`
-  ${inputCss}
+const SearchInputContainer = styled(Row)`
+  background-color: ${({ theme }) => theme.module};
```

Comment:
Why do you depart from inputCss? Instead, should you be changing inputCss?
This is used for other inputs, so it should be used to keep them consistent.

Reply:
yeah works for me